### PR TITLE
Add make_cart_disc

### PR DESCRIPTION
**Pull request by waltsims**
This PR adds make_cart_disc from k-wave v1.4.0 to k-wave-python

### CODE REVIEW

**Comment by waltsims** [comment]
closes #167

File: tests/matlab_test_data_collectors/python_testers/test_make_cart_disc.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        disc_pos = disc_pos.astype(np.float64)
+        radius = float(radius)
+        coordinates = make_cart_disc(disc_pos, radius, focus_pos, num_points, plot_disc, use_spiral)
+        assert np.allclose(coordinates, reader.expected_value_of('coordinates'), equal_nan=True)
+        reader.increment()
+
+    print('make_cart_disc(..) works as expected!')
+
+
+test_make_cart_disc()
```
Comment (faridyagubbayli) [inline tests/matlab_test_data_collectors/python_testers/test_make_cart_disc.py:27]: Can be removed

File: kwave/utils/mapgen.py [hunk 2]
```diff
@@ -10,13 +10,134 @@
...
+        num_points:     Number of points on the disc.
+        plot_disc:      Boolean controlling whether the Cartesian points are plotted (default = False).
+        use_spiral:     Boolean controlling whether the Cartesian points are chosen using a spiral sampling pattern.
+                        Concentric sampling is used by default.
+
+    Returns:
+        disc: 2 x num_points or 3 x num_points array of Cartesian coordinates.
+    """
+    # define literals
+    GOLDEN_ANGLE = 2.39996322972865332223155550663361385312499901105811504
```
Comment (faridyagubbayli) [inline kwave/utils/mapgen.py:40]: Consider moving this to global or better to constants file once you merge `make_cart_bowl (...)` PR.

**Review by faridyagubbayli — APPROVED** [review]
LGTM, left two comments.

**Pull request merged**